.circleci/config.yml

```diff
@@ -249,6 +226,28 @@ jobs:
             docker tag $LOCAL_IMAGE $DOCKER_REPO:latest
             docker push $DOCKER_REPO:latest
 
+  run-end2end-tests:
+    executor: ubuntu-builder
+    working_directory: ~
+    steps:
+      - setup_remote_docker
+      - attach_workspace:
+          at: '~'
+      - run:
+          name: Checkout end2end repo
+          command: |
+            git clone https://github.com/trustlines-network/end2end.git
+      - run:
+          name: Load docker image
+          command: |
+            docker load --input ~/images/$LOCAL_IMAGE.tar
+      - run:
+          name: run end2end tests
+          command: |
+            docker tag $LOCAL_IMAGE $DOCKER_REPO
```

Comment:
Same as in relay, i feel this does not belong here, but more in the load docker image step

Reply:
same reply as in relay: https://github.com/trustlines-network/relay/pull/247#discussion_r250082007